test-case/check-alsabat.sh

```diff
@@ -101,9 +105,9 @@ aplay   -Dplug$pcm_p -d 1 /dev/zero -q || die "Failed to play on PCM: $pcm_p"
 arecord -Dplug$pcm_c -d 1 /dev/null -q || die "Failed to capture on PCM: $pcm_c"
 
 # alsabat test
-# hardcode the channel number of playback to 2, as sof doesnot support mono wav.
-dlogc "alsabat -P$pcm_p --standalone -n $frames -r $rate -c 2 -f $format -F $frequency -k $sigmak"
-alsabat -P$pcm_p --standalone -n $frames -c 2 -r $rate -f $format -F $frequency -k $sigmak & playPID=$!
+# BT offload PCMs also support mono playback.
+dlogc "alsabat -P$pcm_p --standalone -n $frames -r $rate -c $channel_p -f $format -F $frequency -k $sigmak"
+alsabat -P$pcm_p --standalone -n $frames -c $channel_p -r $rate -f $format -F $frequency -k $sigmak & playPID=$!
```

Comment:
the change sounds fine but I can't figure out how this test works with separate commands for playback and capture. I don't have enough understanding to review this.

Reply:
We have some configuration files in framework side to specify PCM IDs and channel numbers, e.g:https://github.com/intel-innersource/drivers.audio.ci.sof-framework/blob/main/parameters/alsabat-playback.csv.

Reply:
>  I can't figure out how this test works with separate commands for playback and capture

Maybe because ALSA man pages are utterly out of date? I struggled with that too, see yesterday's 84dcefd1c9ab57

"man speaker-test" is 11 years old... speaker-test -h looks fine.